index.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Spinnaker&display=swap"
+      rel="stylesheet"
+    />
+    <title>Smashing Heatmap</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="gray-3">
+      <img src="logo.jpg" alt="Smashing" class="header-logo"/>
+      <a href=""></a>
+      <a href=""></a>
+      <a href=""></a>
+      <a href=""></a>
+      <a href=""></a>
+      <a href=""></a>
+      <a href=""></a>
+      <a href=""></a>
+      <button></button>
+      <input type="text" class="header-search"/>
+    </header>
+    <main>
+      <div class="featured-links">
+        <p>Don't Miss These Latest<br />
+        Articles</p>
+        <article class="gray-2">
+          <h2><a href="">A Monthly Update On All Things Smashing</a></h2>
+        </article>
+        <article class="gray-1">
+          <h2><a href="">What Vitruvius Can Teach Us About Web Design</a></h2>
+        </article>
+        <article class="gray-2">
+          <h2><a href="">Solving Common Cross-Platform Issues When Working With Flutter</a></h2>
+        </article>
+        <!-- <article>
+          <a href=""
+            >Mirage JS Deep Dive: Using Mirage JS And Cypress For UI Testing
+            (Part 4)</a
+          >
+        </article> -->
```

Comment:
It’s better to delete **commented piece of code**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))